animations/lib/main.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'src/basics/animation_controller_demo.dart';
+import 'src/misc/expand_card.dart';
+
+void main() => runApp(AnimationSamples());
+
+class Demo {
+  final String name;
+  final String route;
+  final WidgetBuilder builder;
+
+  const Demo(this.name, this.route, this.builder);
+}
+
+final basicDemos = [
+  Demo('Animation Controller', AnimationControllerDemo.routeName,
+      (context) => AnimationControllerDemo()),
+];
+
+final miscDemos = [
+  Demo('Expandable Card', ExpandCardDemo.routeName,
+      (context) => ExpandCardDemo()),
+];
+
+final basicDemoRoutes =
+    Map.fromEntries(basicDemos.map((d) => MapEntry(d.route, d.builder)));
+
+final miscDemoRoutes =
+    Map.fromEntries(miscDemos.map((d) => MapEntry(d.route, d.builder)));
+
+final allRoutes = <String, WidgetBuilder>{
+  ...basicDemoRoutes,
+  ...miscDemoRoutes,
+};
+
+class AnimationSamples extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Animations Samples',
+      theme: ThemeData(
+        primarySwatch: Colors.deepPurple,
+      ),
+      routes: allRoutes,
+      home: HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatelessWidget {
+
```

Comment:
nit: the previous class had no whitespace in between the class declaration and the start of the build method.

Reply:
Now that I'm looking at this, I can see there's no `@override` annotation on the build method. Did you remove it, or was it not added by the IDE?

Reply:
I don't usually use it, since build() and setState() are stable APIs.

> Use the @override annotation judiciously and only for methods where the superclass is not under the programmer's control, the superclass is in a different library or package, and it is not considered stable

I don't see any direction given in the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) - is this documented somewhere?

Reply:
Added for consistency with other samples